pyxem/generators/indexation_generator.py

```diff
@@ -75,6 +77,9 @@ def correlate(self,
             The n orientations with the highest correlation values are returned.
         mask : Array
             Array with the same size as signal (in navigation) or None
+        method : String
+            Name of method used to compute correlation between templates and diffraction patterns. Can be
+            'FastCorrelation' or 'NormalizedCorrelation'.
```

Comment:
pyxem is snake_case, so these should be `'fast_correlation'` and `'normalized_correlation'`